src/app/_variables.scss

```diff
@@ -94,7 +86,9 @@ $easeInOutCubic: cubic-bezier(0.645, 0.045, 0.355, 1);
 }
 
 @mixin draggable-hover-border {
-  outline: 1px solid #ddd;
+  &:hover {
+    outline: 1px solid #ddd;
+  }
```

Comment:
No, this was intentional but I will revert until it's clear if it's needed. 

It was done to enable more than just the hover styles to be customised (default border, hover, focus, active) - I also raised the `@include` up a level on `.item-drag-container`. 

On second thoughts it would probably be better to make each of those interaction styles into their own var (item-border, item-hover, item-focus etc) — will move this back to how it was until it's clearer if/what breakdown of discrete interaction styles are needed